designs/administrator/4.6/Pipelines/index.md

```diff
@@ -0,0 +1,72 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Pipelines
+
+## Description
+The Pipelines pages should be updated on the Admin console to be representative of a view that makes sense for the Admin user. Across all pages:
+ - The Namespace column appears when the All-projects selection has been made by the user.
+ - The kebab on the table functions as it does today.
+ - The Tech Preview badge has now been moved next to the title to make room for the Create button on the  right hand side.
+ - The Create button should be styled with the dropdown as it shows on the Secrets page.
+
+## Pipelines page
+On the Pipelines page the user can navigate between Pipelines, Pipeline Runs, Pipeline Resources, and Conditions via the tabs on the top of the page.
+- From the Create button the user can create Pipelines, Pipeline Resources, and Conditions.
+- The main action the user can take on the Pipelines page is creating a Pipeline from the Pipelines page
+- The Pipelines page contains the Pipelines Name, Last Run, Last Run Status, and Last Started columns
+- If the user selects all-projects from the project selector, the Namespace column appears.
+
+![Pipelines page](img/1-pipelines.jpg)
+
+## Pipeline Runs
+If the user selects all-projects from the project selector, the Namespace column appears. The Pipelines Runs page contains the Name, Status, Started, and Duration columns.
+
+![Pipeline runs](img/2-pipeline-runs.jpg)
+
+## Pipeline Resources
+If the user selects all-projects from the project selector, the Namespace column appears. The main action the user can take on the Pipelines Resources tab is creating a Pipeline Resource from the Pipeline page. The Pipelines Resources page contains the Name, Created, and Type columns.
+
+![Pipeline resources](img/3-pipeline-resources.jpg)
+
+## Condition
+If the user selects all-projects from the project selector, the Namespace column appears. The main action the user can take on the Condition tab is creating a Condition from the Pipelines page. The Pipelines Resources page contains the Name and Created columns
+
+![Condition](img/4-Condition.jpg)
```

Comment:
Ok totally makes sense, do we also want to drop the search across all pages?

Reply:
No, the Search should remain. If they want to find an item, we'd need the search because not everything is rendered into the DOM and thus not something the user can search for with the standard CTRL+F find feature.

Reply:
Ok great, thank you for the details